subprojects/xsts/xsts-analysis/src/test/java/hu/bme/mit/theta/xsts/analysis/XstsTest.java

```diff
@@ -40,6 +44,9 @@
 @RunWith(value = Parameterized.class)
 public class XstsTest {
 
+    private static final String SOLVER_STRING = "Z3";
+    private static final String SMTLIB_HOME = "C:\\smt";
```

Comment:
This windows-specific path should definitely not be hardcoded here. There is default smtlib home defined in the SolverManager, we should use that instead

Reply:
Fixed